<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>777</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>153</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>153</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background/color:#ffff00</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background/color:#ffff00</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>581</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListWidget" name="t1"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="t2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="b3">
           <property name="text">
            <string>Limpiar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b1">
           <property name="text">
            <string>Nueva tarea</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b2">
           <property name="text">
            <string>Eliminar tarea</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo_Archivo"/>
    <addaction name="actionAbrir_Archivo"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_Como"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionCopiar"/>
    <addaction name="actionCortar"/>
    <addaction name="actionPegar"/>
    <addaction name="actionDeshacer"/>
    <addaction name="actionRehacer"/>
   </widget>
   <widget class="QMenu" name="menuTareas">
    <property name="title">
     <string>Tareas</string>
    </property>
    <addaction name="actionNueva_Tarea"/>
    <addaction name="actionEliminar_Tarea"/>
    <addaction name="actionLimpiar"/>
   </widget>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <widget class="QMenu" name="menuMenu_Vertical">
     <property name="title">
      <string>Menu Vertical</string>
     </property>
     <addaction name="actionMostrar"/>
     <addaction name="actionOcultar"/>
    </widget>
    <widget class="QMenu" name="menuMenu_Horizontal">
     <property name="title">
      <string>Menu Horizontal</string>
     </property>
     <addaction name="actionMostrar_2"/>
     <addaction name="actionOcultar_2"/>
    </widget>
    <addaction name="menuMenu_Vertical"/>
    <addaction name="menuMenu_Horizontal"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuTareas"/>
   <addaction name="menuOpciones"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNueva_Tarea"/>
   <addaction name="actionEliminar_Tarea"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionCopiar"/>
   <addaction name="actionCortar"/>
   <addaction name="actionPegar"/>
   <addaction name="actionDeshacer"/>
   <addaction name="actionRehacer"/>
   <addaction name="actionLimpiar"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLimpiar"/>
   <addaction name="actionGuardar"/>
  </widget>
  <action name="actionCopiar">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/copy.png</normaloff>:/iconos/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
   <property name="statusTip">
    <string>Copias el texto seleccionado.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/cut.png</normaloff>:/iconos/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cortar</string>
   </property>
   <property name="statusTip">
    <string>Eliminas y copias el texto seleccionado.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/paste.png</normaloff>:/iconos/paste.png</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
   <property name="statusTip">
    <string>Pegas el texto copiado.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionNuevo_Archivo">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/File.png</normaloff>:/iconos/File.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo Archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAbrir_Archivo">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/open file.png</normaloff>:/iconos/open file.png</iconset>
   </property>
   <property name="text">
    <string>Abrir Archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/save.png</normaloff>:/iconos/save.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionGuardar_Como">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/save.png</normaloff>:/iconos/save.png</iconset>
   </property>
   <property name="text">
    <string>Guardar Como</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNueva_Tarea">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/task.png</normaloff>:/iconos/task.png</iconset>
   </property>
   <property name="text">
    <string>Nueva Tarea</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionEliminar_Tarea">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/cross.png</normaloff>:/iconos/cross.png</iconset>
   </property>
   <property name="text">
    <string>Eliminar Tarea</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionDeshacer">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/undo.png</normaloff>:/iconos/undo.png</iconset>
   </property>
   <property name="text">
    <string>Deshacer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRehacer">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/redo.png</normaloff>:/iconos/redo.png</iconset>
   </property>
   <property name="text">
    <string>Rehacer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionLimpiar">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/broom.png</normaloff>:/iconos/broom.png</iconset>
   </property>
   <property name="text">
    <string>Limpiar </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionMostrar">
   <property name="text">
    <string>Mostrar</string>
   </property>
  </action>
  <action name="actionOcultar">
   <property name="text">
    <string>Ocultar</string>
   </property>
  </action>
  <action name="actionMostrar_2">
   <property name="text">
    <string>Mostrar</string>
   </property>
  </action>
  <action name="actionOcultar_2">
   <property name="text">
    <string>Ocultar</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Recursos/recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionCopiar</sender>
   <signal>triggered()</signal>
   <receiver>t2</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCortar</sender>
   <signal>triggered()</signal>
   <receiver>t2</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPegar</sender>
   <signal>triggered()</signal>
   <receiver>t2</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b3</sender>
   <signal>clicked()</signal>
   <receiver>t2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDeshacer</sender>
   <signal>triggered()</signal>
   <receiver>t2</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRehacer</sender>
   <signal>triggered()</signal>
   <receiver>t2</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLimpiar</sender>
   <signal>triggered()</signal>
   <receiver>t2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMostrar_2</sender>
   <signal>triggered()</signal>
   <receiver>toolBar</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMostrar</sender>
   <signal>triggered()</signal>
   <receiver>toolBar_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOcultar</sender>
   <signal>triggered()</signal>
   <receiver>toolBar_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOcultar_2</sender>
   <signal>triggered()</signal>
   <receiver>toolBar</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
